RetailCoder.VBE/UI/Command/FindAllReferencesCommand.cs

```diff
@@ -163,6 +173,30 @@ private Declaration FindTarget(object parameter)
             return _state.FindSelectedDeclaration(_vbe.ActiveCodePane);
         }
 
+        private Declaration FindFormTarget()
+        {            
+            var project = _vbe.ActiveVBProject;
+            var component = _vbe.SelectedVBComponent;
+            
+            if (component != null && _vbe.SelectedVBComponent.HasDesigner)
+            {
+                var designer = ((dynamic)component.Target).Designer;
```

Comment:
maybe a small note on why this needs a `dynamic` invoke. I think it's rarity justifies a comment